bolt/kv_metrics_test.go

```diff
@@ -0,0 +1,42 @@
+package bolt_test
+
+import (
+	"testing"
+
+	"github.com/influxdata/influxdb/kit/prom"
+	"github.com/influxdata/influxdb/kit/prom/promtest"
+)
+
+func TestInitialKVStoreMetrics(t *testing.T) {
```

Comment:
Hey @zhulongcheng , would you try to port the existing metrics_test.go code here?

I'd like to make sure the metrics are coming out correctly

Reply:
updated